src/components/achievement-card/achievement-card.tsx

```diff
@@ -7,32 +7,36 @@ interface AchievementCardProps {
 
 function UnlockDate (args:{date: number, unlocked: boolean}) {
     if (args.unlocked) {
+        // date is in UTC epoch seconds
+        const localDateTime: Date = new Date(0);
+        localDateTime.setUTCSeconds(args.date);
+
         const formattedDate =  new Intl.DateTimeFormat("en-US", {
             year: "numeric",
             month: "2-digit",
             day: "2-digit",
             hour: "2-digit",
             minute: "2-digit",
             second: "2-digit",
-        }).format(args.date);
+        }).format(localDateTime);
         return <p className={"text-xs"}>{formattedDate}</p>
     }
 }
 
-export default function AchievementCard(props: AchievementCardProps): React.ReactElement {
-    const backgroundColor = props.achievement.unlocked ? 'bg-green-500' : 'bg-gray-500';
+export default function AchievementCard({ achievement: {unlocked, icon, iconGray, name, description, unlockDate} }: AchievementCardProps): React.ReactElement {
+    const backgroundColor = unlocked ? 'bg-green-500' : 'bg-gray-500';
     return (
         <div className={"border-4 flex rounded-md border-blue-400 p-1 " + backgroundColor}>
             <img className={"w-12 h-12"}
-                 src={props.achievement.unlocked ? props.achievement.icon : props.achievement.iconGray}
+                 src={unlocked ? icon : iconGray}
                  alt={"achievement icon"}/>
-            <div className={"flex-col pb-2 pl-1"}>
-                <h3 className="font-bold">{props.achievement.name}</h3>
-                <p className="text-xs">{props.achievement.description}</p>
+            <div className={"flex-col pb-2 pl-1 grow"}>
```

Comment:
This line is the format magic, asks this center div to take up the extra space and pushes the date to the right